spec/paas.yaml

```diff
@@ -1,6 +1,12 @@
 swagger: '2.0'
 info:
-  description: 'Manage your apps using our API'
+  description: "Manage your apps using our API\n\n
+    wss://api.iran.liara.ir\n 
```

Comment:
I guess they also need to append some query parameters like the command and project name to this.

Reply:
Can i access to publish new version to npm?

Reply:
No one has access to publish it except me. I have to fix it so everyone would be able to publish new icons.